src/runtime_src/core/tools/common/ReportCmcStatus.cpp

```diff
@@ -98,8 +98,15 @@ ReportCmcStatus::writeReport( const xrt_core::device* /*_pDevice*/,
   try {
     boost::property_tree::ptree cmc_scale = cmc.get_child("scaling");
     _output << boost::format("  %-22s :\n") % "Runtime clock scaling feature";
-    _output << boost::format("    %s : %s\n") % "Supported" % cmc_scale.get<std::string>("supported");
-    _output << boost::format("    %s : %s\n") % "Enabled" % cmc_scale.get<std::string>("enabled");
+    if (!cmc_scale.get<bool>("supported")) {
+      _output << "    Information unavailable or not applicable" << std::endl;
+      return;
+    }
+    if (!cmc_scale.get<bool>("enabled")) {
+      _output << "    Not enabled" << std::endl;

```

Comment:
```suggestion
      _output << "    Not enabled\n";

```

Reply:
Hi @dbenusov-xilinx , please let me know the reason to replace std::endl with "\n" ?

Reply:
Hi @AShivangi , please let me know the right way to adding new line character. Thanks,

Reply:
I believe our standard practice is not to use std::endl, I heard this from Soren on a previous review.